charts/prometheus-alerts/templates/containers-prometheusrule.yaml

```diff
@@ -1,4 +1,5 @@
 {{ $values              := .Values }}
```

Comment:
This pattern actually is old - I learned later on that even in a `{{ with ...}}` block, you can access `$.Values` and `$.Release` ... so `$` acts as an escape clause. We should just switch to that, and get rid of `$values` and `$release` I think.